tests/db/conftest.py

```diff
@@ -109,11 +109,6 @@ def inputs(user_vk):
 
     # 2. create block with transactions for `USER` to spend
     for block in range(4):
-        transactions = []
-        for i in range(10):
-            tx = b.create_transaction(b.me, user_vk, None, 'CREATE')
-            tx_signed = b.sign_transaction(tx, b.me_private)
-            transactions.append(tx_signed)
-
+        transactions = [signed_create_tx for _ in range(10)]
```

Comment:
From what I saw of the `Block`, it doesn't care that each of the transactions are the same, but in this fixture it does seem a bit odd that they're all the same. To me, it'd make more sense if a new transaction was created each time.